### DISCUSSION TITLE
How to retry action in stackstorm workflow

### DISCUSSION THREAD

**Discussion by komalphadatare** [StackStorm/st2#5481] [Q&A]
hi,
I want to retry core.remote action how we can achieve retry action in workflow

I have tried below format but it shows me error tasks['xyz'] - should NOT have additional properties ("retry")

version: 1.0

input:
  - command

tasks:
  task1:
    action: core.remote cmd=<% ctx().command %>
    retry:
      delay: 1
      count: 3
    next:
      - when: <% succeeded() %>
        do: task2

  task2:
    action: core.noop

Regards,
komal

**Comment by minsis** [comment]
Without seeing how your [yaml is formatted](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code) its really difficult to troubleshoot this.

Have you looked at the [workflow examples](https://github.com/StackStorm/st2/tree/master/contrib/examples/actions/workflows/)? Specifically [this one](https://github.com/StackStorm/st2/blob/master/contrib/examples/actions/workflows/orquesta-task-retry.yaml) about retries

Without the formatting I can only make guesses here. It seems like maybe you have the key `retry` in the incorrect place:
```yaml
tasks:
  my__task:
  retry:
```

Whereas it should be under `my_task`:
```yaml
tasks:
  my_task:
    retry:
```
